### DISCUSSION TITLE
2023.5.19 签到失败

### DISCUSSION THREAD

**Discussion by tctitan** [qd-today/templates#336] [General]
2023-05-20 09:10:35
日志：Failed at 1/27 request,
Fail assert: {"re": "error_code\":\"0\"", "from": "content"} from success_asserts,
Response Error : HTTP 599: Connection timed out after 30000 milliseconds,
Request URL: https://api.smzdm.com/v1/user/checkin

**Comment by a76yyyy** [comment]
599错误不是框架错误, 属于超时响应错误, 一般为目标网站问题或者自己容器所在的网络问题, 具体请百度 `http 599 超时错误`
